server/handlers/authentication.go

```diff
@@ -27,7 +27,7 @@ func Login(w http.ResponseWriter, r *http.Request) {
 			}
 		}
 
-		utils.ParseHTML(w, "login.html",
+		utils.ParseHTML(w, "assets-v1/templates/src/pages/oauth_portal/login.html",
```

Comment:
I think it makes sense to extract path to the html file into a const variable and reuse it.

Reply:
Yes, I also agree, will do that.